packages/interface/src/utils/fetchTally.ts

```diff
@@ -0,0 +1,39 @@
+import { config } from "~/config";
+
+import type { Tally } from "./types";
+
+import { createCachedFetch } from "./fetch";
+
+const cachedFetch = createCachedFetch({ ttl: 1000 * 60 * 10 });
+
+export interface GraphQLResponse {
+  data?: {
+    tally: Tally;
+  };
+}
+
+const tallyQuery = `
+  query Tally {
+    tally (id: $id) {
```

Comment:
I think @0xmad mentioned here should be modified as 
```
query Tally {
    tally(id: $id) {
```
remove the space between `tally` and `(id` since that's the format we fetch the polls.